packages/pcsx-rearmed/build.sh

```diff
@@ -0,0 +1,23 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/notaz/pcsx_rearmed
+TERMUX_PKG_DESCRIPTION="Yet another PCSX fork based on the PCSX-Reloaded project"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=23
+TERMUX_PKG_SRCURL=https://github.com/notaz/pcsx_rearmed/archive/refs/tags/r${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=887e9b5ee7b8115d35099c730372b4158fd3e215955a06d68e20928b339646af
+TERMUX_PKG_DEPENDS="libpng, pulseaudio, zlib"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_post_get_source() {
+	wget -O - https://github.com/rtissera/libchdr/archive/a03e69319164f69d781ab8e453f8cf407387bd13.tar.gz | tar -zxf -
+	wget -O - https://github.com/notaz/libpicofe/archive/33787db41d955f8dcafe833097f2cc87d70186ec.tar.gz | tar -zxf -
```

Comment:
Use `termux_download` function to download extra files from internet. An example exists in <a href="https://github.com/termux/termux-packages/blob/e3a1bd1d46bbad317613ce5e22fa4c0e1e217862/packages/docbook-xsl/build.sh">`build.sh` of `docbook-xsl` package</a>.

Or you can alternatively specify multiple values to `TERMUX_PKG_SRCURL`. An example is in 
<a href="https://github.com/termux/termux-packages/blob/e3a1bd1d46bbad317613ce5e22fa4c0e1e217862/packages/ncurses/build.sh">`build.sh` of `ncurses` package</a>.

Also some comment is suggested about how to update these sources. Currently this implies that we can update them to HEAD in the main branch. Is that correct?

Reply:
Ah, maybe they are just Git submodules? Then you can specify Git repository as `TERMUX_PKG_SRCURL` (beginning with `git+`), as in <a href="https://github.com/termux/termux-packages/blob/e3a1bd1d46bbad317613ce5e22fa4c0e1e217862/packages/megacmd/build.sh">`megacmd`</a>.

Reply:
I think it's better to keep up with the commits provided by the developer, so I'll just go with what Leonid suggested. I have `TERMUX_PKG_SRCURL` pointing to the commits returned by `git submodule`.

Reply:
> I think it's better to keep up with the commits provided by the developer,

The `git+[...]` URL handles that correctly. See the scripts for details. But no problem with the current solution downloading tarballs for submodules, except for maitainablity.

Reply:
I did consider to use `git+[...]` at first, but I'm experiencing an issue on my side; `curl: (1) Protocol "git+https" not supported or disabled in libcurl`.
Should I ignore and push anyway?

Reply:
curl is not supposed to be used for `git+[...]` URL, which should be handled by `termux_git_clone_src`. See 20bd0f0dc099aa390778f832f0201d50814b54ee for reference.